draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -335,6 +335,19 @@ When considering a proposed congestion control algorithm, the community MUST
 consider the following criteria. These criteria will be evaluated in various
 domains (see {{general-use}} and {{special-cases}}).
 
+Some criteria below will use the keyword SHOULD for some or all of their
+content, generally because there are practical limitations on carrying out an
+evaluation. In these cases, the SHOULD supersedes the MUST above.
```

Comment:
Do we really say "SHOULD supersedes the MUST"? i.e. is there no other way to formulate this -- if the community goes to evaluate some criteria and decides there's a practical limitation on doing so, does that not count as having considered it along the lines of "MUST consider the following criteria"?

Just curious if there's a better way to structure this so that we don't have the potential for confusion about a SHOULD being stronger than a MUST.

Reply:
I don't want to wade too deeply into semantics, but what I'm thinking about is those undocumented RT CCs. If we quickly decide to do nothing, have we "considered" it? I suppose that weakens the MUST if that counts as considering.

That said, I'm not married to this wording and am happy to take an alternative, including the original wording!

Reply:
I really dislike using MUST and SHOULD as nouns. That's jargon. How about:

Some of the sections below will list criteria that SHOULD be met. It may happen that
these criteria are not in fact met by the proposal. In such cases, the community
MUST document whether not meeting the criteria is acceptable, for example
because there are practical limitations on carrying out an evaluation of the criteria.

That is, specify that proposals may well not meet a criteria, and that it is the working group responsibility to assess whether that's OK before forwarding the document for publication.